packages/react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -168,6 +176,7 @@ export class GroupedListBase extends React.Component<IGroupedListProps, IGrouped
             usePageCache={usePageCache}
             onShouldVirtualize={onShouldVirtualize}
             version={version}
+            renderEarly={true}
```

Comment:
I'm a bit concerned about changing this default behavior as doing so broke `TilesList` (which is why it's not enabled by default now). This change will also affect `DetailsList` as `GroupedList` is used internally for rendering in that control in "grouped" mode. I know we strive to be accessible by default so I like this change in principle -- how confident are you that enabling this won't break `GroupedList` usage?

Reply:
I suppose it's impossible to really be sure, since GroupedList seems to get some pretty custom usage.

The only other potential approach I can think of would be to manually apply `tabIndex={0}` to the first GroupHeader, though that's equally buggy since it wouldn't help folks using custom GroupHeaders. It also seems potentially a little less reliable than letting the FocusZone take care of applying tabIndex.

Thoughts on which is better?

Reply:
Maybe the answer here to allow people to opt out of early rendering?

`renderEarly={typeof props.renderEarly === 'boolean' ? props.renderEarly : true}`

(perhaps there is a less verbose way of writing that).

The early rendering is a better option as it addresses the underlying issue of the DOM just not being present but it is also much more likely to break something using `GroupedList` as it changes the rendering behavior. `tabIndex` is better in that it is highly unlikely to break rendering.

Allowing folks to opt out will at least give people an easy change should they need it. That said, there is an argument that this is a breaking change and we should rather require users to opt in.

Not a serious suggestion: `GroupedListV3`: now with early rendering? 

... actually that last suggestion got me thinking: is this an issue with `GroupedListV2`? The rendering is quite different so it may not be (though it still ultimately uses `List`).